Makefile

```diff
@@ -489,6 +493,10 @@ e2e-akeyless:
 e2e-gcp:
 	bats -t test/bats/gcp.bats
 
+.PHONY: e2e-alibabacloud
+e2e-alibabacloud:
```

Comment:
Add `ALIYUNCLI` as a dependency for this target

Reply:
> Add `ALIYUNCLI` as a dependency for this target

updated, thanks!